docs/classic-ui/page-templates/index.md

```diff
@@ -0,0 +1,320 @@
+
+
+#### z3c.jbot
+
+
+`z3c.jbot` (https://pypi.org/project/z3c.jbot/ ) can override page templates (``.pt`` files) for views, viewlets, old style page templates and portlets. In fact, it can override any ``.pt`` file in the Plone source tree.
+
+To override a particular file, first determine its canonical filename. It’s defined as the path relative to the package within which the file is located; directory separators are replaced with dots.
+
+As a simple example, to edit the default search page in Plone Classic we first find the existing template. In a fresh plone install, it's in the Products.CMFPlone  `/Products/CMFPlone/browser/templates/search.pt`  so our override file will be called `Products.CMFPlone.browser.templates.search.pt`
+
+Create a new directory inside your local plone install for template overrides and copy the system `search.pt` into that directory, and rename it to  `Products.CMFPlone.browser.templates.search.pt`
+
+```
+├─ PloneSite
+│  ├─ bin
+│  ├─ eggs
+│  │   └─Products.CMFPlone-version
+│  │     └Products
+│  │      └CMFPlone
+│  │       └browser
+│  │         └templates
+│  │           └search.pt  <--- override this 
+│  ├─ parts
+│  └─ mysite
+      ├─ configure.zcml
+      └─ template-overrides 
+         └─ Products.CMFPlone.browser.templates.search.pt <--- with this
+```
+
+In your configure.zcml, register the `template-overrides` directory with jbot
+
+```xml+genshi
+<include package="z3c.jbot" file="meta.zcml" />
+
+<browser:jbot
+    directory="template-overrides"
+    />
+```
+
+Now, edit your `Products.CMFPlone.browser.templates.search.pt` and see your changes on the plone search page.
+
+```{note}
+   The `<browser:jbot>` zcml should also contain a `layer` attribute for best practice.  see the z3c.jbot documentation and browser layers documentation
+```
+
+#### overriding with zcml
+
+Although not best practice, you can also quickly override templates by creating an `override.zcml` and adding your custom registration.  For the search page, this would be finding the ZCML entry for the search view in `Products/CMFPlone/browser/configure.zcml` and copying it into your own `overrides.zcml`, but change the `template=` attribute to point to your custom template.  The `class` also must change to become an absolute path to the original class path
+
+overrides.zcml
+```xml+genshi
+ <browser:page
+      name="search"
+      class="Products.CMFPlone.browser.search.Search"
+      permission="zope2.View"
+      for="plone.app.layout.navigation.interfaces.INavigationRoot"
+      template="template-overrides/search.pt"
+      />
+```
+
+#### layering with zcml
+
+A better and much more flexable way to override templates, especially when developing add-ons, is to use a browser layer in your configure.zcml.  This is *highly* preferred to using an overrides.zcml file, but involves using a BrowserLayer Interface.  This is extremely easy and is best practice.
```

Comment:
```suggestion
A better and much more flexible way to override templates, especially when developing add-ons, is to use a browser layer in your configure.zcml.  This is *highly* preferred to using an overrides.zcml file, but involves using a BrowserLayer Interface.  This is extremely easy and is best practice.
```

Reply:
Done in [f3dca39](https://github.com/plone/documentation/pull/1139/commits/f3dca39172c68f931da703e4691c66dd900ca013)